### ISSUE TITLE
[Tooltip]: accessibility issues related to the component

### ISSUE THREAD

**Issue by NatalliaAlieva** [epam/UUI#1498]
### Description
Tooltip text "Some text" is not available to keyboard users

### Precondition
Screen Reader is **OFF**

### Steps to Reproduce
1. Open [AvatarStack](https://uui.epam.com/documents?category=components&id=avatarStack&mode=doc&skin=UUI4_promo) component.
2. Use the `Tab` key to navigate to the image avatars with tooltips and observe.

### Actual Result
Tooltip text "Some text" is not available using keyboard.

https://github.com/epam/UUI/assets/57751283/9ccf8d51-85ea-4fc4-b013-380781c315d1

### Expected Result
Tooltip text "Some text" should be available using keyboard, avatar images should be focused using `Tab` key from the keyboard. When avatar image is focused, tooltip text should opens automatically. 
**Failed WCAG Checkpoint:** 2.1.1 Keyboard (Level A)
https://www.w3.org/WAI/WCAG21/Understanding/keyboard.html

### Description
Tooltip text "Some text" disappears when the pointer moved over it

### Precondition
Screen Reader is **OFF**

### Steps to Reproduce
1. Open [AvatarStack](https://uui.epam.com/documents?category=components&id=avatarStack&mode=doc&skin=UUI4_promo) component.
2. Hover mouse pointer over Tooltip button -> tooltip text "Some text" is displayed.
3. Hover mouse pointer over tooltip text "Some text"  and observe.

### Actual Result
Tooltip text "Some text" disappears when the pointer moved over it.

https://github.com/epam/UUI/assets/57751283/f0873c08-cdcb-4fb0-897e-72e3fd2f0805

### Expected Result
The pointer can be moved over the tooltip text without the tooltip text disappearing.
**Failed WCAG Checkpoint:** 1.4.13 Content on Hover or Focus (Level AA)
https://www.w3.org/WAI/WCAG21/Understanding/content-on-hover-or-focus.html

**Comment by vasilii-kovalev** [comment]
Seems related to #2322.
